test/integration/controlplane/transformation/kms_transformation_test.go

```diff
@@ -626,7 +626,7 @@ resources:
 		// Need to enable this explicitly as the feature is deprecated
 		featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.KMSv1, true)
 
-		test, err := newTransformTest(t, encryptionConfig, false, "", nil)
+		test, err := newTransformTest(t, transformTestConfig{transformerConfigYAML: encryptionConfig, runtimeConfig: []string{"api/alpha=true", "api/beta=true"}})
```

Comment:
no need to change it, but isn't this equivalent to `api/all=true`?

Reply:
Yes, symmetry with feature gates again (https://github.com/kubernetes/kubernetes/pull/130325/files/56bd8d86a58d6060741ca8a8bd9873e77087d5d9#r1971050990).